sql/wsrep_schema.cc

```diff
@@ -739,6 +739,14 @@ int Wsrep_schema::store_view(THD* thd, const Wsrep_view& view)
   Wsrep_schema_impl::binlog_off binlog_off(thd);
   Wsrep_schema_impl::sql_safe_updates sql_safe_updates(thd);
 
+  bool trans_started= false;
+  if (trans_begin(thd, MYSQL_START_TRANS_OPT_READ_WRITE))
+  {
+    WSREP_ERROR("Failed to start transaction for store view");
+    goto out;
+  }
```

Comment:
We do not really need the variable `trans_started`. The `goto out` could simply be replaced with `DBUG_RETURN(ret)` or a new `out_not_started:` label added right before the final statement of the function if we prefer `goto out_not_started` to duplicating the `DBUG_RETURN(ret)`.

Reply:
Fixed.